DirectEmployers/MyJobs
# [PD-2149] Create jira ticket when new access request is submitted

This builds on top of #2088 by ensuring that a ticket is created when the request is submitted. I ended up writing a new functioon (that doubles as a task) to facilitate creating a new ticket, as the original wasn't well documented, did some extra email type things, and couldn't be run as a task. I couldn't simply wrap a task around it as that would have caused merge conflicts anyway. I suspect that this new task could deprecate that one (`log_to_jira`), but I didn't want to risk breaking old functionality to find out.

# Testing
As before:
- head to /request-company-access
- enter a company name and submit the form
- head over to [Jira](https://directemployersfoundation.atlassian.net/projects/MEMBERSUP/issues/MEMBERSUP-810?filter=allopenissues) in the MEMBERSUP project and note that a new ticket should have been created.
